wolox-books/src/app/screens/user/user.service.ts

```diff
@@ -16,4 +16,9 @@ export class UserService {
     const data = Object.assign({ user: {...user, locale: 'en'} });
     return this.httpClient.post(`${this.URL}/api/v1/users`, data, {observe: 'response'});
   }
+
+  login(login: any): Observable<HttpResponse<Object>> {
+    const data = Object.assign({ session: { ...login } });
```

Comment:
```suggestion
    const data = { session: { ...login } };
```

`Object.assign` is util when you have a target and a source object. But, this time you only need to create a new object.